Soulgram.Interests/Soulgram.Interests.Application/Queries/GenGenresIdsByNameQuery.cs

```diff
@@ -0,0 +1,47 @@
+﻿using MediatR;
+using Soulgram.Interests.Application.Interfaces;
+using Soulgram.Interests.Domain;
+
+namespace Soulgram.Interests.Application.Queries;
+
+public class GenGenresIdsByNameQuery : IRequest<Dictionary<string, string?>?>
+{
+    public GenGenresIdsByNameQuery(ICollection<string>? names)
+    {
+        Names = names;
+    }
+
+    public ICollection<string>? Names { get; }
+}
+
+public class GenGenresIdsByNameQueryHandler : IRequestHandler<GenGenresIdsByNameQuery, Dictionary<string, string?>?>
```

Comment:
you can use it into GenGenresIdsByNameQuery and name it just like 'internal sealed class Handler'

Reply:
got it, i will